engine/access/subscription/streamer.go

```diff
@@ -89,7 +95,14 @@ func (s *Streamer) sendAllAvailable(ctx context.Context) error {
 		response, err := s.sub.Next(ctx)
 
 		if err != nil {
-			if errors.Is(err, storage.ErrNotFound) || errors.Is(err, storage.ErrHeightNotIndexed) || execution_data.IsBlobNotFoundError(err) {
+			if errors.Is(err, ErrResponseNotAvailableForBlock) {
+				continue
+			}
+
+			if errors.Is(err, storage.ErrNotFound) ||
```

Comment:
I think this is where we need to take a step back and consolidate error handling. An action for a future issue.